src/site/resources/layout.php

```diff
@@ -28,7 +28,7 @@
 </head>
 <body>
 <h1>
-    <a href="/">PHPMD - PHP Mess Detector</a>
+    <a href="<?php echo $baseHref ?? ''; ?>/">PHPMD - PHP Mess Detector</a>
```

Comment:
That uses PHP 7 syntax. Preferably we shouldn't do that throughout the "old code".

Reply:
The website generator does not need to have the same compatibility than the library. The site is not even in the production package. Before it was even built by Java, so here we just have to say you need PHP 7 to build the website. There are already PHP 7-only code in the website builder, this is not the first one.

Reply:
Can we seperate the website generater from phpmd core code?

Reply:
We currently decided to have it in the same repository so if we have a new feature we can update the documentation in the same PR.
It can be an idea to move the site directory out of the src directory. But that should be another PR.

Reply:
```suggestion
    <a href="<?php echo isset($baseHref) ? $baseHref : ''; ?>/">PHPMD - PHP Mess Detector</a>
```
Should be php 5 capable no?